schema/objects/StockPlan.schema.json

```diff
@@ -29,6 +29,10 @@
       "description": "The initial number of shares reserved in the pool for this stock plan by the Board or equivalent body.",
       "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/types/Numeric.schema.json"
     },
+    "cancellation_behavior": {
```

Comment:
We should consider naming this `default_cancellation_behavior` if we allow it to be overridden on a transaction level (see PR #381)